src/share/poudriere/html/assets/poudriere.js

```diff
@@ -866,559 +824,557 @@ function process_data_jail(data) {
+    window.location.pathname.lastIndexOf('/') + 1,
   );
-  if (pathname == "") {
-    page_type = "index";
+  if (pathname === '') {
+    pageType = 'index';
   } else {
-    page_type = pathname.substr(0, pathname.length - 5);
+    pageType = pathname.substr(0, pathname.length - 5);
   }
 
-  if (page_type == "build") {
-    if (server_style == "hosted") {
-      page_mastername = getParameterByName("mastername");
-      page_buildname = getParameterByName("build");
-      if (!page_mastername || !page_buildname) {
-        $("#loading p")
-          .text("Invalid request. Mastername and Build required.")
-          .addClass("error");
+  if (pageType === 'build') {
+    if (serverStyle === 'hosted') {
+      pageMasterName = getParameterByName('mastername');
+      pageBuildName = getParameterByName('build');
+      if (!pageMasterName || !pageBuildName) {
+        $('#loading p')
+          .text('Invalid request. Mastername and Build required.')
+          .addClass('error');
         return;
       }
-      data_url = "data/" + page_mastername + "/" + page_buildname + "/";
-      $("a.data_url").each(function () {
-        var href = $(this).attr("href");
-        $(this).attr("href", data_url + href);
+      dataURL = `data/${pageMasterName}/${pageBuildName}/`;
+      $('a.data_url').each(() => {
+        const href = $(this).attr('href');
+        $(this).attr('href', dataURL + href);
       });
-      $("#master_link").attr("href", jail_url(page_mastername));
-    } else if (server_style == "inline") {
-      $("#master_link").attr("href", "../");
-      $("#index_link").attr("href", "../../");
+      $('#master_link').attr('href', jailURL(pageMasterName));
+    } else if (serverStyle === 'inline') {
+      $('#master_link').attr('href', '../');
+      $('#index_link').attr('href', '../../');
     }
-    setup_build();
-  } else if (page_type == "jail") {
-    if (server_style == "hosted") {
-      page_mastername = getParameterByName("mastername");
-      if (!page_mastername) {
-        $("#loading p")
-          .text("Invalid request. Mastername required.")
-          .addClass("error");
+    setupBuild();
+  } else if (pageType === 'jail') {
+    if (serverStyle === 'hosted') {
+      pageMasterName = getParameterByName('mastername');
+      if (!pageMasterName) {
+        $('#loading p')
+          .text('Invalid request. Mastername required.')
+          .addClass('error');
         return;
       }
-      data_url = "data/" + page_mastername + "/";
-      $("a.data_url").each(function () {
-        var href = $(this).attr("href");
-        $(this).attr("href", data_url + href);
+      dataURL = `data/${pageMasterName}/`;
+      $('a.data_url').each(() => {
+        const href = $(this).attr('href');
+        $(this).attr('href', dataURL + href);
       });
-      $("#latest_url").attr("href", build_url(page_mastername, "latest"));
-    } else if (server_style == "inline") {
```

Comment:
Why remove `server_style` support? It's not clear in the commits what's going on here. Restyling the file and removing logic makes it hard to review. I'm starting to think of the XZ hack with these large changes. Make them reviewable please.
See dfe3002d411f3d5e5419c7c95aab5dff567dfcea

Reply:
`server_style` variable has been renamed to `serverStyle` following camal case convention. (most other variable names too) Right now, `server_style` is defined in each html file before including js scripts, but this is a bad practice since we have to edit three files to change one variable. So I defined it in the beginning of `poudriere.js` and renamed `server_style` as `serverStyle`

Please check line 30 of modified poudriere.js